### PR TITLE
test: added tests for createContext

### PR DESCRIPTION
**Pull request by anuraghazra**

### CODE REVIEW

File: src/utils/utils.test.tsx [hunk 1]
```diff
@@ -1,11 +1,15 @@
+import React from "react";
+import MockDate from "mockdate";
+import { render } from "reakit-test-utils";
+import { valueToPercent } from "@chakra-ui/utils";
```
Comment (navin-moorthy) [inline src/utils/utils.test.tsx:4]: import valueToPercent from out utils itself.

**Review by navin-moorthy — APPROVED** [review]
LGTM 🎉

File: src/utils/utils.test.tsx (hunk 1 above)
Comment (anuraghazra) [reply src/utils/utils.test.tsx:4]: Opps.

Comment (anuraghazra) [reply src/utils/utils.test.tsx:4]: VSCode auto imported

**Pull request merged**